### DISCUSSION TITLE
get force in Output station

### DISCUSSION THREAD

**Discussion by kmcivil** [FEALiTE/FEALiTE2D#14] [General]
Hi dear salah
thank you for you fantastic application. it is really valuable
I have a question. how can i get forces on beam in specific place?

**Comment by SaMohamed** [comment]
Hi!,
you can use this method `GetElementInternalForcesAt` if the force is in a load case https://github.com/FEALiTE/FEALiTE2D/blob/db1aabac8fd559adebec5083a38de41af487381d/FEALiTE2D/Structure/PostProcessor.cs#L437

 or use this method if the force is in a load combination https://github.com/FEALiTE/FEALiTE2D/blob/db1aabac8fd559adebec5083a38de41af487381d/FEALiTE2D/Structure/PostProcessor.cs#L457
